pattern-filter/apps/pattern-filter/src/main/java/com/redhat/training/transform/TransformRouteBuilder.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.training.transform;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+import org.springframework.context.annotation.Bean;
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+import org.apache.camel.component.jms.JmsComponent;
+import javax.jms.JMSException;
+
+//import org.apache.camel.model.dataformat.XmlJsonDataFormat;
```

Comment:
We should provide students with all the required imports. Specially if there is no step in the GE that indicates to add this dependency, and we can't rely on IDEs to complete this for us.